src/plugins/ClausePlugin.js

```diff
@@ -365,7 +373,10 @@ function ClausePlugin(customLoadTemplate, customParseClause, clauseProps) {
     fromHTML,
     renderToolbar,
     renderAnnotation,
-    rewriteClause
+    rewriteClause,
+    queries: {
```

Comment:
Why is this inside `queries`?

Reply:
This is so we will be able to use it as a query on the editor. So we can use it like `editor.findClauseNode('some-clause-id')`.
Slate has a few built in, but you can create custom ones too.
https://docs.slatejs.org/guides/commands-and-queries#reusing-commands-and-queries

Reply:
Are the rest of these also custom? `renderAnnotation`, `renderToolbar`, `fromHTML`, etc? I can't find most of them in the docs, but if they're custom why aren't they also in `queries`?

Reply:
No, those are custom, but they are not queries on the editor. `Queries are similar to commands, but instead of manipulating the current value of the editor, they return information about the current value, or a specific node, etc.` In the case of `findClauseNode`, we are just returning a clause node, so `queries` is the correct place for this one to go.